### ISSUE TITLE
setting image size in markdown

### ISSUE THREAD

**Issue by andronovhopf** [iterative/blog#70]
I'm not able to change the size of images using markdown. For example:

```
![](/uploads/images/img.png =250x250)
```
seems to throw an error and no image displays (only a little icon). To my knowledge, this is the markdown syntax for resizing images?

**Comment by fadilmamedov** [comment]
@shcheklein can you please assign this to me?

**Comment by frontexpert** [comment]
@shcheklein I looked to solution for issue but now the plugin that this project is using has not this option of resizing the image into markdown! 

How we can see https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-images/src/__tests__/index.js

My proposal to a solution to this issue is:

- 1º Switch  `![](/image.ext)` to `<img src=''/image.ext' class='custom-class-dvc' width='200' height='200'>` and create a custom calss css to adjust
> but this is no better solution because we lose the beneficite of gatsby-image and gatsby-shape plugins

How we can see here in the screenshot
![Screenshot_2020-01-28_07-46-37](https://user-images.githubusercontent.com/19707900/73257458-921b6e00-41a2-11ea-8a48-3df44ad7c06d.png)
 ---

- 2º Switch transformer MD to MDX in this way we get the better of between about markdown and React
We'll can create a component React to the custom image 

```markdown
![test image size](/uploads/images/2017-05-15/git-server-or-github.jpeg)
```

```JSX
<ImageDvc width='200' height='200' position='left' > 
```

> but this solution is very hard because this is can break anything in this blog  and we'll need with this

What do you think about this? Do you have another solution to this issue? tell me

**Comment by shcheklein** [comment]
Don't like both options, tbh.

Can we write our own custom plugin for this? The same way we do for the "external-link" now, for example.

**Comment by frontexpert** [comment]
OK, I will try to find out the way just like "external-link". Thanks.

**Comment by frontexpert** [comment]
@shcheklein I did custom the gatsby-remark-images plugin.
Please take a look at my result.
![Screenshot at Jan 30 12-25-36](https://user-images.githubusercontent.com/19707900/73441541-d3a34900-435b-11ea-9f2e-aedf6792887e.png)

You can append `=WIDTHxHEIGHT` after Title Text to resize the image.
ex:
`![](./pic/pic100.png "Img Title =100x200")`
You can skip the HEIGHT
`![](./pic/pic100.png "Img Title =100")`

**Comment by shcheklein** [comment]
fixed by #92
